components/constellation/pipeline.rs

```diff
@@ -530,13 +529,40 @@ impl UnprivilegedPipelineContent {
         Ok(child_process)
     }
 
+    // gaol is not supported on Windows yet, so this is just multiprocess
+    // without sandboxing
```

Comment:
The reason I didn't do that is that we don't have the `gaol` crate available on Windows at all.  `cfg!(...)` evaluates to a boolean, so the types would still need to be available for both paths through the function.  Is there a way to do this that I'm not seeing?  The only other way I could think of was to create separate spawn_sandboxed() and spawn_unsandboxed() methods that spawn_multiprocess calls, and then have a windows-only spawn_sandboxed that panics.

With my latest changes at https://github.com/servo/ipc-channel/pull/108, multiprocess Servo works on Windows.